motoman_motopos_d500_support/urdf/motopos_d500_macro.xacro

```diff
@@ -60,9 +62,16 @@
     <axis xyz="0 0 -1" />
     <limit lower="-3.4906" upper="3.4906" effort="392" velocity="2.7925"/>
   </joint>
-  <joint name="${prefix}joint_2-tool0" type="fixed">
-    <origin xyz="0 0 0.150 " rpy="0 0 0"/>
-    <parent link="${prefix}link_2"/>
+
+  <joint name="${prefix}joint_2-flange" type="fixed">
+    <origin xyz="0 0 0.150" rpy="0 0 0"/>
+    <parent link="${prefix}link_1"/>
```

Comment:
This says it goes from link 2 to flange (which is correct) but it actually goes from link 1 to flange.